plot_goes_x.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+import argparse
```

Comment:
System imports (argparse) in one block, 3rd party in the next block, then finally local package imports (not applicable here).

Reply:
Thanks!  And it is always hard to know how much time to take to improve code.  This looks like new code but is just the plot_hrc.py code because I didn't care and I didn't think it was worth generalizing.  Overall I probably should have just left this PR as  WIP while soak testing.